api/envoy/extensions/filters/http/basic_auth/v3/basic_auth.proto

```diff
@@ -41,6 +41,12 @@ message BasicAuth {
   // If it is not specified, the username will not be forwarded.
   string forward_username_header = 2
       [(validate.rules).string = {well_known_regex: HTTP_HEADER_NAME strict: false}];
+
+  // This field specifies the header name to load basic authentication credentials.
+  //
+  // If it is not specified, the filter loads credential from ``:Authorization`` header
+  string authentication_header = 3
```

Comment:
How about naming it `credential_header`?  It seems more precise than `authentication_header` .

Reply:
I actually like `authentication_header` since it indicates it is carrying data needed for authentication.

Reply:
I suggested an alterantive name because the original header is `authorization`, and naming this field as `authentication_header` might create some confusion. It's not a strong opinion though.

Reply:
Authorization header is used to carry authentication information is usual case in practice. So, I am fine to the authentication_header, but credentical_header is also ok for me. It up to you guys.